docs/hsi.md

```diff
@@ -402,14 +446,17 @@ References:
 BootGuard is a processor feature that prevents the machine from running firmware images not released by the system manufacturer.
 It forms a root-of-trust by fusing in cryptographic keys into the processor itself that are used to verify the Authenticated Code Modules found in the SPI flash.
 
-Possible results:
+**Impact:** When BootGuard is not set up correctly then the chain-of-trust between the CPU and the bootloader can not be verified.
+This would allow subverting the Secure Boot protection.
```

Comment:
I believe that any time you say "allow subverting the Secure Boot protection" you could go one step further. What can the attacker do when "Secure Boot protection is subverted"?

Reply:
`This would allow subverting the Secure Boot protection which gives the attacker full access to your hardware.`?

Reply:
Sounds good.

Reply:
Fixed, thanks.